### PR TITLE
Improve debug diagnostics

### PR DESCRIPTION
**Pull request by FUDCo**
Various debugging, logging, and diagnostic improvements prior to landing resolved promise pruning.

### CODE REVIEW

File: packages/SwingSet/src/kernel/kernel.js [hunk 1]
```diff
@@ -710,6 +724,7 @@ export default function buildKernel(kernelEndowments) {
     }
 
     kernelKeeper.setInitialized();
+    commitCrank(); // commit "crank 0"
```
Comment (warner) [inline packages/SwingSet/src/kernel/kernel.js:727]: This is an (important, but) unrelated bugfix.. let's land this all together now, but next time maybe let's put this into a separate PR

File: packages/SwingSet/src/kernel/vatManager.js [hunk 2]
```diff
@@ -116,7 +111,9 @@ export default function makeVatManager(
     insistCapData(args);
     // TODO: disable send-to-self for now, qv issue #43
     const target = mapVatSlotToKernelSlot(targetSlot);
-    kdebug(`syscall[${vatID}].send(vat:${targetSlot}=ker:${target}).${method}`);
+    const argList = legibilizeMessageArgs(args).join(', ');
```
Comment (warner) [inline packages/SwingSet/src/kernel/vatManager.js:114]: is there an easy way to avoid the perf hit when kdebug is disabled?

I'm ok with landing this as-is, but I'd like a ticket to remind us to speed it up in the next few weeks.

File: packages/SwingSet/test/test-transcript-light.js [hunk 3]
```diff
@@ -15,6 +15,8 @@ async function testLoadState(t, withSES) {
   config.hostStorage = storage;
   const c = await buildVatController(config, withSES, ['one']);
   const state0 = getAllState(storage);
+  t.equal(state0.initialized, 'true');
```
Comment (warner) [inline packages/SwingSet/test/test-transcript-light.js:18]: this would go with the `commitCrank` bugfix PR, if that were a separate PR

**Review by warner — APPROVED** [review]
Looks good

File: packages/SwingSet/src/kernel/vatManager.js (hunk 2 above)
Comment (FUDCo) [reply packages/SwingSet/src/kernel/vatManager.js:114]: Filed as #925
